style.css

```
@@ -228,3 +229,69 @@ body {
     width: 20px;
     margin-right: 10px;
 }
+
+footer .information-footer {
+    display: flex;
+    justify-content: space-between;
+    color: #42446E;
+    font-family: DM Sans;
+    font-size: 18px;
+    font-style: normal;
+    font-weight: 400;
+}
+
+footer .information-footer .infor {
```

Se vc aplicar a mudança que, falei a cima vc pode remover esse selector.
```suggestion
.information-footer .infor {
```

---

feito!